libkbfs/rekey_queue.go

```diff
@@ -57,7 +61,9 @@ func (rkq *RekeyQueueStandard) Enqueue(id tlf.ID) <-chan error {
 			// spawn goroutine if needed
 			var ctx context.Context
 			ctx, rkq.cancel = context.WithCancel(context.Background())
-			go rkq.processRekeys(ctx, rkq.hasWorkCh)
+			for i := 0; i < numRekeyWorkers; i++ {
```

Comment:
Good point; Thanks! I'm not sure we can simply replace the queue with `chan` here, since there's a `IsRekeyPending` method which needs to iterate through the queue. I suppose we could use a separate `map` to track pending ones, which should also make the code more readable. Will try to refactor this a little bit